src/components/widgets/Map.vue

```diff
@@ -437,15 +434,28 @@ watch(home, () => {
   const position = home.value
   if (position === undefined) return
 
-  if (homeMarker.value === undefined) {
-    homeMarker.value = L.marker(position as LatLngTuple)
-    const homeMarkerIcon = L.divIcon({ className: 'marker-icon', iconSize: [32, 32], iconAnchor: [16, 16], html: 'H' })
-    homeMarker.value.setIcon(homeMarkerIcon)
-    const homeMarkerTooltip = L.tooltip({ content: 'No data available', className: 'waypoint-tooltip' })
+  if (!homeMarker.value) {
+    homeMarker.value = L.marker(position as LatLngTuple, {
+      icon: L.divIcon({ className: 'marker-icon', iconSize: [24, 24], iconAnchor: [12, 12] }),
+      draggable: true,
+    })
+    const homeMarkerTooltip = L.tooltip({
+      content: '<i class="mdi mdi-home-map-marker text-[18px]"></i>',
+      permanent: true,
+      direction: 'center',
+      className: 'waypoint-tooltip',
+      opacity: 1,
+    })
     homeMarker.value.bindTooltip(homeMarkerTooltip)
+    homeMarker.value.on('dragend', (e: L.DragEndEvent) => {
+      const marker = e.target as L.Marker
+      const latlng = marker.getLatLng()
+      setHomePosition([latlng.lat, latlng.lng])
+    })
     map.value.addLayer(homeMarker.value)
+  } else {
+    homeMarker.value.setLatLng(position as LatLngTuple)
```

Comment:
With this logic, I understand the home position will be set, and will be there on the map, automatically, but won't be used as the vehicle home, right? This will probably raise a lot of questions from the user. We should just completely differentiate this vehicle home from the original map home, which can be called something else, otherwise we will be causing unnecessary confusing to the user, as we are already in the code.